crates/bevy_core_pipeline/src/prepass/mod.rs

```diff
@@ -37,6 +38,7 @@ use bevy_utils::FloatOrd;
 
 pub const DEPTH_PREPASS_FORMAT: TextureFormat = TextureFormat::Depth32Float;
 pub const NORMAL_PREPASS_FORMAT: TextureFormat = TextureFormat::Rgb10a2Unorm;
+pub const VELOCITY_PREPASS_FORMAT: TextureFormat = TextureFormat::Rg32Float;
```

Comment:
Future optimization, we could probably use an Rg16Float texture here? Not sure how widely supported that format is though.

Reply:
I want to use Rg32Float to be safe for now, but I added a TODO to investigate using Rg16Float.